pkg/machinery/client/client.go

```diff
@@ -532,8 +532,10 @@ func (c *Client) Bootstrap(ctx context.Context, req *machineapi.BootstrapRequest
 }
 
 // Shutdown implements the proto.MachineServiceClient interface.
-func (c *Client) Shutdown(ctx context.Context) (err error) {
-	resp, err := c.MachineClient.Shutdown(ctx, &emptypb.Empty{})
+func (c *Client) Shutdown(ctx context.Context, force bool) (err error) {
```

Comment:
we might need to add option arguments here to maintain backwards compatibility, I can take care of that

Reply:
did the update